src/spikeinterface/core/analyzer_extension_core.py

```diff
@@ -405,9 +412,13 @@ def _run(self, verbose=False, **job_kwargs):
             self._compute_and_append_from_waveforms(self.params["operators"])
 
         else:
-            for operator in self.params["operators"]:
-                if operator not in ("average", "std"):
-                    raise ValueError(f"Computing templates with operators {operator} needs the 'waveforms' extension")
+            bad_operator_list = [
+                operator for operator in self.params["operators"] if operator not in ("average", "std")
+            ]
+            if len(bad_operator_list) > 0:
```

Comment:
maybe if `any(bad_operator_list)` is slightly more readable (?)

Reply:
I think that is stylistic. I don't use `any` that often in code I think checking the length of the list is just my default. I don't know what others think?

Reply:
I think it's clear enough :)